Src/Sarif.PartnerMatcher.Function/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
you could remove this and update the .gitignore from the root folder

Reply:
deleted